### PR TITLE
quincy: doc/ceph-volume: refine "prepare" top matter

### PR DESCRIPTION
**Pull request by zdover23**
This commit refines the top matter in the
doc/ceph-volume/lvm/prepare.rst file.

Signed-off-by: Zac Dover <zac.dover@gmail.com>
(cherry picked from commit 233af44cb4d4b0a83a1f32b893febb49540aefc3)

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/ceph-volume/lvm/prepare.rst [hunk 1]
```diff
@@ -2,25 +2,24 @@
 
 ``prepare``
 ===========
-This subcommand allows a :term:`filestore` or :term:`bluestore` setup. It is
-recommended to pre-provision a logical volume before using it with
-``ceph-volume lvm``.
+This subcommand facilitates the setup of :term:`filestore` or
+:term:`bluestore`. Before you run ``ceph-volume lvm prepare``, we recommend
+that you provision a logical volume. Then you can run ``prepare`` on that
+logical volume.
```
Comment (anthonyeleven) [inline doc/ceph-volume/lvm/prepare.rst:8]: The filestore / bluestore bit is improperly capitalized, but I suggest removing as it isn’t relevant.  “of OSDs”.

File: doc/ceph-volume/lvm/prepare.rst [hunk 2]
```diff
@@ -2,25 +2,24 @@
...
-
-:term:`LVM tags` makes volumes easy to discover later, and help identify them as
-part of a Ceph system, and what role they have (journal, filestore, bluestore,
-etc...)
-
-Although :term:`bluestore` is the default, the back end can be specified with:
+``prepare`` uses :term:`LVM tags` to assign several pieces of metadata to a
+logical volume. Volumes tagged in this way are easier to identify and easier to
+use with Ceph. :term:`LVM tags` identify logical volumes by the role that they
+play in the Ceph cluster (for example: journal, filestore, or bluestore).
```
Comment (anthonyeleven) [inline doc/ceph-volume/lvm/prepare.rst:19]: With filestore deprecated, maybe not mention it or journal.  (for example: BlueStore data, BlueStore WAL+DB)

**Review by anthonyeleven — APPROVED** [review]

File: doc/ceph-volume/lvm/prepare.rst (hunk 1 above)
Comment (zdover23) [reply doc/ceph-volume/lvm/prepare.rst:8]: See https://github.com/ceph/ceph/pull/48665.

File: doc/ceph-volume/lvm/prepare.rst (hunk 2 above)
Comment (zdover23) [reply doc/ceph-volume/lvm/prepare.rst:19]: See https://github.com/ceph/ceph/pull/48665.

**Pull request merged**